### DISCUSSION TITLE
Automatic syncing stream when switching it.

### DISCUSSION THREAD

**Discussion by julpegor** [f1multiviewer/issue-tracker#332] [Feature requests] [answered]
If I switch the stream on any video (replay mode, had no way to check it on live races), I get back to the start, even if I sinced this window already. Would be nice to set the new stream timestamp to the one I just switched.

**Comment by M0UL** [comment, marked as answer]
duplicate of #330

**Reply by julpegor** [reply]
Sorry you're right. Just searched for "syncinc stream" and "switching stream" issues.
